app/src/main/java/app/revanced/integrations/youtube/utils/VideoUtils.java

```diff
@@ -104,12 +104,20 @@ public static boolean inAppDownloadButtonOnClick(String videoId) {
     }
 
     public static void launchExternalDownloader() {
-        launchExternalDownloader(VideoInformation.getVideoId());
+        launchExternalDownloader(VideoInformation.getVideoId(), false);
     }
 
-    public static void launchExternalDownloader(@NonNull String videoId) {
+    public static void launchExternalDownloader(@NonNull String resourceId, boolean isPlaylist) {
```

Comment:
We can seperate this function into `launchVideoExternalDownloader` and `launchPlaylistExternalDownloader`